src/avatar_action.h

```diff
@@ -57,7 +57,7 @@ bool fire( avatar &you, map &m );
  */
 bool fire( avatar &you, map &m, item &weapon, int bp_cost = 0 );
 // Throw an item  't'
-void plthrow( avatar &you, int pos = INT_MIN,
+void plthrow( avatar &you, item_location &loc,
```

Comment:
Why does this require a non-const `item_location` reference, wouldn't a const reference suffice? Alternatively, `item_location` is copyable, so a value would work as well.

Reply:
a const ref will not suffice, but i'll look into the alternative